### ISSUE TITLE
Remote Displays (PFD, ND, ECAM)

### ISSUE THREAD

**Issue by mspann** [flybywiresim/a32nx#6712]
### Aircraft Version

Development

### Description

I tried out the Remote MCDU using the server.exe. It is a great idea and works smoothly for me.
I would like to see the rest of the displays implemented in this way.

Use Case: cockpit builders and users who would like to get a display displayed on an tablet.

**Comment by ryucz** [comment]
+Upvote

**Comment by Rascal650** [comment]
I second this request and I would love to see ND on a remote display as well (a la MCDU) as I always have the ND window popped out (R ALT + click).  I understand that the EFB is going to have this capability soon but for me personally, the ND (and PFD) are more important.   How can we bump the priority of this request?

**Comment by ErickSharp** [comment]
> I second this request and I would love to see ND on a remote display as well (a la MCDU) as I always have the ND window popped out (R ALT + click). I understand that the EFB is going to have this capability soon but for me personally, the ND (and PFD) are more important. How can we bump the priority of this request?

You can’t, the team works at our own pace. However, a comprehensive solution is being worked on already to allow for all displays to be remotely accessed.

**Comment by Rascal650** [comment]
That's great news!  Thanks for sharing

> > I second this request and I would love to see ND on a remote display as well (a la MCDU) as I always have the ND window popped out (R ALT + click). I understand that the EFB is going to have this capability soon but for me personally, the ND (and PFD) are more important. How can we bump the priority of this request?
> 
> You can’t, the team works at our own pace. However, a comprehensive solution is being worked on already to allow for all displays to be remotely accessed.

That's great news!!  Thanks for sharing.

**Comment by Benjozork** [comment]
The solution will allow every display to be run over the network but it still requires a lot of integration and optimization. 

Here's a preview.

![unknown-2174(1).png](https://user-images.githubusercontent.com/4503241/162888463-1f0cf3b9-98da-4188-a2d1-590f3ce024c5.png)

**Comment by sfo** [comment]
Any news on this one? 

@Benjozork already showed a prototype 1,5 year ago. So I was wondering, if there is some kind of alpha/beta release available that we may be able to contribute to instead of doing work in parallel or even duplicate?

We are currently evaluating options to move from our legacy system (X-Plane 10 based model + [Extplane](https://github.com/vranki/ExtPlane)  + custom display implementation) to a modern setup with MSFS and this A320 model.
We are just wondering if it is worth to wait for this feature or start our own implementation, which may result in an adaption of the current Extplane and display software.

**Comment by tracernz** [comment]
I wouldn't make any plans based on this feature coming anytime soon. If you would like to and have the ability to contribute that would of course be welcomed, but otherwise your alternate solution is a safer bet.

**Comment by robin24** [comment]
Hey there,

Just wanted to revive this issue to add a +1 to this effort.

I am a fully blind flight sim pilot, and have been actively engaged in this hobby for around 6 years at this point.
Developers in our community have created an addon called [Talking Flight Monitor](https://talkingflightmonitor.com) which enables us to gain access to various aircraft systems and panels of aircraft by using SDKs and APIs provided by plane developers. Currently, we have this fully implemented for PMDG's Boeing lineup, allowing us to conduct full IFR flights, with full access to the FMC, MCP and pretty much all instrument panels.

We would love to eventually expand TFM to support the really amazing FBW fleet as well, and being able to remotely gain access to systems such as the Flypad would be a huge step towards making this possible.

For example, the remote MCDU which can already be accessed via SimBridge is quite usable for us, the layout is a bit tricky to navigate using a screenreader, however with some good understanding of how everything is designed it is very much doable.
It would be absolutely fantastic if the Flypad could be exposed in the same way, as currently, having looked at the developer docs it appears there is no API in place which would allow for external applications to access and manipulate its contents.

Given how central the Flypad is in setting up the aircraft, e.g. importing SimBrief flight plans and setting fuel / payload, this would be a major step enabling us to hopefully build some really solid accessibility support for it.

In case some of you would like to learn more about how blind flight simming works, we have a [Community Wiki](https://bvipilots.net) with lots of info and videos.
Also, I did start a conversation about this very issue on the FBW Discord in the SDK channel, so feel free to reach out to me there with any questions.

**Comment by gagagu** [comment]
Hi, I am using msfs for mixead reality but popped out panels using with SpaceDesk on my iPads are not working while in VR and ASOBO haven't fixed this for long time. The only way to use you plane in Mixed Reality is to have all dieplays in remote like MCDU.
It would be really nice to have this.

thx forward

**Comment by FrancescoTalotta** [comment]
Hello,
Over the past few days, I have been working on rendering the PFD in a web browser by streaming SimConnect data from one computer to another. The results are quite promising and represent an important step toward home cockpit implementations of the FBW project, as well as toward supporting alternative platforms (e.g. Linux) as remote clients.

<img width="1654" height="1354" alt="Image" src="https://github.com/user-attachments/assets/e4d40856-099b-4382-985d-0290afe2032e" />

**Comment by Drachenkaetzchen** [comment]
> Hello, Over the past few days, I have been working on rendering the PFD in a web browser by streaming SimConnect data from one computer to another. The results are quite promising and represent an important step toward home cockpit implementations of the FBW project, as well as toward supporting alternative platforms (e.g. Linux) as remote clients.

That's awesome. Is there any place to test this?

**Comment by FrancescoTalotta** [comment]
Hello, I will try to push the code on my github with instructions in the upcoming weeks.
